formasaurus/classifiers.py

```diff
@@ -260,7 +319,7 @@ def classify_proba(self, form, threshold=0.0):
         return self._probs2dict(probs, threshold)
 
     def train(self, annotations):
-        """Train FormExtractor on a list of FormAnnotation objects."""
+        """Train formtype_model on a list of FormAnnotation objects."""
```

Comment:
Not sure the new wording is accurate, but there is no `FormExtractor` class, so I strongly suspect that the previous wording was out of date.